src/Service.Tests/SqlTests/GraphQLPaginationTests/MySqlGraphQLPaginationTests.cs

```diff
@@ -41,11 +41,11 @@ public static async Task SetupAsync(TestContext context)
             DisplayName = "Test after token for long values.")]
         [DataRow("long_types", 1, "", 1, "", true,
             DisplayName = "Test after token for long values for last page.")]
-        [DataRow("string_types", "\"\"", "\"null\"", 1, 3, false,
+        [DataRow("string_types", "\"\"", "\"lksa;jdflasdf;alsdflksdfkldj\"", 1, 2, false,
```

Comment:
`"\"lksa;jdflasdf;alsdflksdfkldj\""` is an end cursor value?

Reply:
nvm. I see it here: 
![image](https://github.com/Azure/data-api-builder/assets/6414189/9d9b27f3-2aec-4643-979b-94c157e6d35e)